R/bayesian/model_details_panel.R

```diff
@@ -76,29 +106,27 @@ model_details_panel_ui <- function(id) {
 #' @param id ID of the module
 #' @param model Reactive containing bayesian meta-analysis for all studies
 #' @param model_sub Reactive containing meta-analysis with studies excluded
-model_details_panel_server <- function(
-    id,
-    model,
-    model_sub
-    ) {
+model_details_panel_server <- function(id, models) {
   moduleServer(id, function(input, output, session) {
     ns <- session$ns
+    
+    main_model <- models[[1]]
```

Comment:
It can, but only the final tab will use the rest of the models. This is the form of the existing app. I can check with the rest of the team whether that is correct though.